### PR TITLE
Add support for using .babelrc in the app root.

### PR DESCRIPTION
**Pull request by arunoda**
Reverts https://github.com/zeit/next.js/pull/466
And allows to configure babel via the `.babelrc` file.

If there's no such file, we'll add our preset `next/babel` by default. If not we'll use the `.babelrc` file.
Most of the time, user need to add `next/babel` as a preset.

If we are taking this in, we could support ES2015 in the `config.next.js` as well.

### CODE REVIEW

**Comment by rauchg** [comment]
Missing `README` ref

**Comment by arunoda** [comment]
@rauchg Oh! I forgot that :)

**Comment by arunoda** [comment]
@rauchg done.

File: README.md [hunk 1]
```diff
@@ -458,28 +458,20 @@ module.exports = {
...
-  }
+{
+  "presets": [
+    "next/babel",
+    "stage-0"
+  ],
 }
 ```
 
+Although it's not required, you will need to add `next/babel` as a preset. That's the default preset we use for all Next.js apps.
```
Comment (rauchg) [inline README.md]: so, we removed support for `babel` in `next.config.js` right? Otherwise, we also have to make a mention of that here

Comment (arunoda) [inline README.md]: We removed that. Now there's no such option.

**Comment by rauchg** [comment]
@arunoda needs re-base 🙏

**Comment by arunoda** [comment]
@rauchg done.

**Pull request merged**